data/fetch.mjs

```diff
@@ -326,7 +326,7 @@ function fetchRewardsProvenance(ws) {
   return ws.queryLedgerState("rewardsProvenance")
 }
 
-async function fetchDReps(ws, { stakePools }) {
+async function fetchDReps(ws, stakePools = {}) {
```

Comment:
Is `stakePools` parameter ever provided? Following your change in `query` call I don't see it anymore.